client/app/views/TrackViewer/Sidebar.vue

```diff
@@ -143,7 +154,15 @@ export default defineComponent({
           @track-split="$emit('track-split', $event)"
           @track-seek="$emit('track-seek', $event)"
           @update-new-track-settings="$emit('update-new-track-settings',$event)"
```

Comment:
This can probably be removed because you have the same event emitting under the `<creation-mode>` component.